setup.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
I'm not sure I understand why this file needs to be included, the only steps that should be contained in a script are 

```
rails db:create db:schema:load db:create_admin
```

which would be better suited as a rake task that just invokes those other 3 tasks. 

Postgres should be provided by the postgres container, ruby should be provided by the ruby container, and the gem install should be done as a step in the container so that it can be actually cached. There's a common pattern used for Rails applications with Docker where you copy your Gemfile and Gemfile.lock into the Dockerfile first, install your dependencies, and then copy in the rest of the app in order to cause less cache misses, https://github.com/mitre/heimdall/blob/master/Dockerfile#L12-L20

Reply:
My idea with the setup.sh script was to have it be able to set up both a standard install and a docker install, which is why the Postgresql and rvm install are in there. I've moved them to a different argument now. 

Wrt the rakes, I'll make one that's just db:initial_setup. I'll also look into copying the gemfile over first,

Reply:
The biggest issue I have with the `setup.sh` script is it is not platform agnostic, it is tied specifically to Debian Linux, I don't really want you to put in the work to make it compatible with other platforms though, since the Docker container kinda does that for us without any dedicated bash scripts.